testing/kuttl/scripts/pgbackrest-initialization.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+EXPECTED_STATUS=$1
+
+CLUSTER=${CLUSTER:-default}
+
+REPO_POD=$(kubectl -n ${NAMESPACE} get pod --selector=postgres-operator.crunchydata.com/cluster=${CLUSTER},postgres-operator.crunchydata.com/pgbackrest-dedicated --no-headers -o custom-columns=":metadata.name")
+
+# Grab the status line, remove whitespace with `xargs`
+STATUS=$(kubectl -n ${NAMESPACE} exec -i ${REPO_POD} -c pgbackrest -- pgbackrest info |\
+    grep status |\
+    xargs)
```

Comment:
⛏️ I suspect the backslashes aren't necessary here. `grep` and `xargs` are running next to `kubectl`, right?